src/core/operations/UnicodeTextFormat.mjs

```diff
@@ -0,0 +1,67 @@
+/**
+ * @author Matt C [me@mitt.dev]
+ * @copyright Crown Copyright 2020
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation.mjs";
+import Utils from "../Utils.mjs";
+
+/**
+ * Unicode Text Format operation
+ */
+class UnicodeTextFormat extends Operation {
+
+    /**
+     * UnicodeTextFormat constructor
+     */
+    constructor() {
+        super();
+
+        this.name = "Unicode Text Format";
+        this.module = "Default";
+        this.description = "Adds Unicode combining characters to change formatting of plaintext.";
+        this.infoURL = "https://en.wikipedia.org/wiki/Combining_character";
```

Comment:
Remember to remove the localisation (en.) from wikipedia urls. Removing it means that users get given their local version (if it exists).